packages/upload/src/vaadin-upload.js

```diff
@@ -103,7 +104,9 @@ class Upload extends ElementMixin(ThemableMixin(PolymerElement)) {
       <slot name="file-list">
         <ul id="fileList" part="file-list">
           <template is="dom-repeat" items="[[files]]" as="file">
-            <vaadin-upload-file file="[[file]]" i18n="[[i18n]]"></vaadin-upload-file>
+            <li>
```

Comment:
Also moved the `<li>` element under the same scope with the `<ul>` (for semantic correctness).